terasoluna-gfw-parent/pom.xml

```diff
@@ -344,6 +344,14 @@
       </dependency>
       <!-- == End MyBatis == -->
 
+      <!-- == Begin Hibernate == -->
+      <dependency>
+        <groupId>org.hibernate</groupId>
+        <artifactId>hibernate-core</artifactId>
+        <version>${org.hibernate.version}</version>
+      </dependency>
```

Comment:
Why don't you have the same version of `hibernate-entitymanager`?
I think... considering the users who are still referencing via HEM, misalignment will cause confusion.

Reply:
add dependencyManagement definition for entity-manager.
8508d2eed071c61f267993f2dc8b33d4f6f426ac in #1025